float64 data1
float64 data2
---
bool result
